rails-app/app/controllers/audios_controller.rb

```diff
@@ -25,21 +21,30 @@ def new
 
   # GET /audios/1/edit
   def edit
+    @audio = set_audio
   end
 
   # POST /audios
   # POST /audios.json
   def create
```

Comment:
Lots of stuff going in there, comment your method using http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md 

# Converts the object into textual markup given a specific format.
#
# @param format [Symbol] the format type, `:text` or `:html`
# @return [String] the object converted into the expected format.
def to_format(format = :html)
  # format the object
end